src/Presentation/GraphQL/Base/GraphSchema.cs

```diff
@@ -0,0 +1,17 @@
+using GraphQL.Types;
+
+namespace Presentation.GraphQL.Base
+{
+    public class GraphSchema : Schema
+    {
+        public GraphSchema()
+        {
+            Mutation = new Mutation();
+            Mutation.Name = "caca";
```

Comment:
Should we use here a better naming like for example "mutation" ?

Reply:
Done